File: Level 1/약수의 합/약수의 합.py
def sumDivisor(num):
    answer = 1
    divider = 2
    while divider <= num:
        if num % divider == 0:
            answer += divider
            divider += 1
        else:
            divider += 1
    return answer

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(sumDivisor(12))

File: Level 1/약수의 합/약수의 합.cpp
#include<iostream>
using namespace std;

int sumDivisor(int n)
{
	int Divider = 1;
	int Sum = 0;
	while (Divider <= n) {
		if (n % Divider == 0) {
			Sum += Divider;
		}
		Divider++;
	}
	return Sum;
}

int main()
{
	int testCase = 10;
	int testAnswer = sumDivisor(testCase);

	cout << testAnswer;
}